src/content/docs/style-guide/writing-strategies/contribute-to-site.mdx

```diff
@@ -0,0 +1,376 @@
+
+import styleguideManageProjects from 'images/style-guide_screenshot-crop_edit-manage-project-status.webp'
+
+import styleguideGHDbranch from 'images/style-guide_screenshot-crop_edit-ghd-new-branch.webp'
+
+import styleguideBaseCompare from 'images/style-guide_screenshot-crop_edit-ghd-base-compare.webp'
+
+
+The docs team welcomes your contributions, whether you're a New Relic employee or a New Relic user. This doc walks you through three different methods for making or requesting changes to the docs site. 
+
+## Prerequisites [#prereq]
+
+Whatever method you choose, you need to sign up for a [GitHub account](https://github.com/join) if you haven't yet. 
+
+GitHub is at the heart of our whole process. When you propose a change to the docs site, GitHub manages the two different versions of a doc: the original version as it exists on docs.newrelic.com and the edited version that exists on your screen. This creates a protective layer that allows the docs team to review and approve changes before they're released to the site. 
+
+<Callout variant="tip">
+If you're a New Relic employee, we recommend joining the New Relic org on GitHub. To join the org, ping the @hero at #help-opensource on Slack. 
+</Callout>
+
+## Contribute to the docs site [#contribute-work]
+
+You don't need a complete understanding of git or GitHub to get set up, but you might encounter some unfamiliar terms:   
+
+* A **commit** acts like a save. 
+* A **branch** houses your commit(s).
+* A **pull request** packages your branch and sends it to us for review. 
+
+<Tabs>
+	<TabsBar>
+        <TabsBarItem id="edit-browser">
+            Edit via the browser
+        </TabsBarItem>
+        <TabsBarItem id="copy-repo">
+            Copy the site to your GitHub account
+        </TabsBarItem>
+        <TabsBarItem id="request">
+            Make a request to the team 
+        </TabsBarItem>
+  </TabsBar>
+
+    <TabsPages>
+        <TabsPageItem id="edit-browser">
+
+For most cases, we recommend editing the docs site via your browser. On every page, you can click **Edit this doc** to edit directly from your browser. 
+
+<Steps>
+    <Step>
+    ### Click **Edit this doc** 
+
+On the doc you'd like to edit, click **Edit this doc** from the right nav. This opens a text view of the doc in question, letting you view the copy alongside its code. 
+    
+  <img
+    title="Make small changes to a doc"
+    alt="Click the edit button to make small changes"
+    src={styleguideEditDoc}
+  />
+
+    </Step>
+    <Step>
+        ### Make your edits 
+
+Once you've opened the text-version of a doc, find the **pencil** icon in the top nav of the text element. This lets you interract with the doc in edit-mode rather than the default view-only mode. 
+
+    <img
+    title="Click the pencil button"
+    alt="Click the pencil button to go into edit mode"
+    src={styleguidePencilButton}
+    /> 
+
+Make your edits, and don't worry too much about formatting or grammar. We've got that covered.
```

Comment:
Can we make this more friendly? 

"Make any changes you like. Don't worry about formatting or grammar. That's what your Docs team is here for." Or something like that